src/components/views/my-covid-alerts.tsx

```diff
@@ -133,7 +133,7 @@ export const MyCovidAlerts = () => {
               <BubbleIcons.Info fill={colors.purple} width={56} height={57} />
             }
             padding={{r: 10}}>
-            <Text style={text.largeBlack}>
+            <Text style={text.largeBold}>
```

Comment:
Is there a separate issue for these changes?

Reply:
Used to be https://github.com/project-vagabond/covid-green-app/issues/279, I squeezed them in... 😬